src/app/shared/services/account/account.service.ts

```diff
@@ -321,7 +326,6 @@ export class AccountService {
       .pipe(map((response: AuthResponse) => {
         if (response.isSuccessful) {
           this.setAccount(response.getAccountVO());
-          this.setArchive(response.getArchiveVO());
```

Comment:
It is related! This `setArchive` call is actually pointless since no ArchiveVO is attached to this specific response. When logging in, it doesn't do anything, but when verifying MFA as part of this change, this sets the current archive to `null` which then logs the user out!

Reply:
Thank you!